source/includes/items/_create.md

```diff
@@ -0,0 +1,342 @@
+## Create An Item
+
+> `POST http://localhost:3000/api/v6/items`
+
+```ruby
+example_example_params = {
+  item: {
+    location_id: Integer,
+    condition_id: Integer,
+    status_id: Integer,
+    code: String,
+    usage: String ['in_use', 'unused'],
+    project: Text,
+    x: Decimal, 
+    y: Decimal,
+    purchase_date: Date, 
+    purchase_price_cents: Decimal,
+    room: String,
+    warranty: Integer, 
+    warranty_info: Text,
+    action_id: Integer,
+    installation_date: Date, 
+    life_expectancy: Float,
+    catalogue_item_id: ???,
+    condition_details: String,
+    
+    photos_attributes: [
+      {
+        image: String ???
+      }
+    ],
+    
+    task_attributes: [ ??? remove? ???
```

Comment:
should we just hide this in the docs?

Reply:
- Not sure about the photo likely a @pacMakaveli question
- Yeh, remove task attributes